pyblish_lite/control.py

```diff
@@ -95,7 +95,7 @@ def on_next():
 
         util.defer(100, on_next)
 
-    def emit_(self, signal, kwargs):
+    def emit_(self, signal, **kwargs):
```

Comment:
Oh, no, sorry! I meant it's *good* that you stay consistent with QML. I was just explaining why QML wasn't doing this already, because otherwise it should have happened there as well.

Do make it a `kwargs` like it was please.